cura/PrinterOutput/PrintJobOutputModel.py

```diff
@@ -147,4 +151,14 @@ def updateState(self, new_state):
 
     @pyqtSlot(str)
     def setState(self, state):
-        self._output_controller.setJobState(self, state)
\ No newline at end of file
+        self._output_controller.setJobState(self, state)
+
+    @pyqtProperty("QVariantList", notify=configurationChangesChanged)
+    def configurationChanges(self) -> List[ConfigurationChangeModel]:
```

Comment:
If this is behavior that you really, really want in here, I think it's better to create a ClusterPrintJobOutputModel and put it in there.

Reply:
I did discuss this with Ruben and he seemed ok with it. Most of the plugin code will be replaced sometime in the near future, but I can see how it would be worth the effort of keeping stuff out of Cura core where possible.

A subclass of `PrintJobOutputModel` sounds like a good solution.